backend/LexBoxApi/GraphQL/CustomTypes/ProjectMembersVisibilityMiddleware.cs

```diff
@@ -18,7 +18,9 @@ public async Task InvokeAsync(IMiddlewareContext context, IPermissionService per
             if (!await permissionService.CanViewProjectMembers(projId))
             {
                 // Confidential project, and user doesn't have permission to see its users, so only show the current user's membership
-                context.Result = projectUsers.Where(pu => pu.User?.Id == loggedInContext.MaybeUser?.Id).ToList();
+                context.Result = projectUsers.Where(pu =>
+                    pu.User?.Id == loggedInContext.MaybeUser?.Id ||
```

Comment:
huh, I know this was already here. But if User is null, and MaybeUser is null, then both left and right will be null, meaning this is included when maybe it shouldn't be? I'm not sure.

Reply:
I think that's a good catch.